packages/api/tests/models/test-providers-model.js

```diff
@@ -149,6 +149,44 @@ test('Providers.create() encrypts the credentials using KMS', async (t) => {
   );
 });
 
+test('Providers.create() does not encrypt already encrypted credentials', async (t) => {
```

Comment:
How does this test prove that credentials aren't re-encrypted? By the fact that they can be successfully decrypted at the end?